### ISSUE TITLE
Request: more granular partitioning for LMDE

### ISSUE THREAD

**Issue by Ammako** [linuxmint/linuxmint#395]
It would be nice to have more granular options for partitioning, at least with LMDE 5 later. The installer is too restrictive with the kind of partitioning setups you can do.

You can't even set up encryption outside of the automatic partitioning, it would be nice if one could have more direct control over that. The automatic partitioning is nice for beginners who don't really want to have to tinker with it, but it would be nice if you could for instance partition your disk as `/boot/efi` + encrypted root (including /boot), this is possible on regular Debian installer, and on most other distros as well, but on lmde if you select manual partitioning then you can't even have any encryption at all, unless you're advanced enough to be able to add encryption post-install.

Ubuntu's installer for example allows you to select a logical volume for installation, if you set up lvm with a volume group and some logical volumes, and open them before starting up the installer, your logical volumes under /dev/mapper/ are recognized by the installer, and you can install directly to them, which at least makes the task easier. I'd imagine regular Mint can do that too, because their installers are essentially the same. On LMDE though, your logical volumes just don't show up on the manual partitioning.

Those are just some examples, but is there a reason why you couldn't just adapt Calamares for LMDE instead of rolling your own and much more limited installer? Since Mint essentially uses the same installer as Ubuntu's and benefits from the same functionality, would be nice if LMDE could be the same as well, but with Debian's installer.

I understand that Calamares isn't the most user-friendly for beginners, yours is a lot more beginner-friendly which is good, but the manual partitioning at least could use having more granular control. If you're going to be manually partitioning then you should be assumed to be a relatively advanced user to begin with, so from a beginner-friendly point of view I don't think manual partitioning being more powerful would hurt things too much.
